### ISSUE TITLE
NPC Sheet needed

### ISSUE THREAD

**Issue by 9littlebees** [superseva/outgunned#4]
I've run a campaign with this module and honestly it works great in all respects except one area - NPC sheets. Please consider adding this. All it needs is a grit tracker with customisable flame markers, a place to write their attack and defence values, and a place to drag and drop evemy feats and special actions (from items).

In my campaign I typed these all out into Journals but it was a lot of extra work, especially since I had already created all the enemy feats and special actions as items.

**Comment by superseva** [comment]
I feel you, but atm it's hard to find the time to do it. The npc sheets should be flexible enough to at least have grit tracker configurable. 
Unfortunately the only way atm for this to be continued is for someone to send PR or fork it.

**Comment by KarstenW76** [comment]
The Problem is, that the NPC sheets are not working. After clicking on create -> NPC, an actor seems to be created, but it cannot be opened up.
This is the console info to be found, nothing happens. If you could probably take a quick test? Probably it's only a small edit for you without longer searching...

<img width="553" alt="Bildschirmfoto 2024-05-18 um 16 43 06" src="https://github.com/superseva/outgunned/assets/65510738/ff61a41d-37ab-439b-8b1f-966934ffcad8">

**Comment by superseva** [comment]
That is because the NPC is only defined in the system.json as existing entity. But I never got to develop sheet for it. I never even released this on Foundry. It was just a roller that we wanted for our game. You can say I accidentally developed Character sheet.

**Comment by KarstenW76** [comment]
Ah, OK, understood, this makes it clear ... thanks for clarification. So if you find the time it would be great, otherwise we're glad you did what you did and made it even playable ... and it works fine with what it got :)

**Comment by superseva** [comment]
I know the pain, to be honest we wanted just to try the game but we soon went back to play usual suspects (swade and myz). For me Outgunned was fun but somehow 'math' and dice results didn't click with our party. 

Btw I think I should be more clear on git repo. I did update the Readme file saying the dev and maintenance is halted for now

**Comment by KarstenW76** [comment]
OK, thank you, this is fully understandable. So probably someone could possibly take it over for further improvement. I'll put this question in the Discord.

**Comment by 9littlebees** [comment]
Indeed, to echo Karsten, I'm very grateful you created this module and got so much functionality running. The fact that the only thing missing is an NPC sheet, and even then there is a workaround (journal entries and the Global Progress Clocks module) is really quite great. Hopefully someone in the community will fork the module and carry on your good work!

**Comment by 9littlebees** [comment]
Oops, accidentally closed the issue, but it should remain open so that anyone coming to fork the project can see this.
